composer.json

```diff
@@ -18,16 +18,16 @@
         }
     ],
     "require": {
-        "php": "^8.0",
-        "illuminate/contracts": "^8.0|^9.0",
-        "illuminate/filesystem": "^8.0|^9.0",
-        "illuminate/support": "^8.0|^9.0",
-        "illuminate/console": "^8.0|^9.0",
-        "illuminate/routing": "^8.0|^9.0",
-        "storyblok/php-client": "^1.6",
-        "storyblok/richtext-resolver": "^2.1",
+        "php": "^8.1",
+        "ext-json": "*",
         "apix/cache": "^1.3",
-        "ext-json": "*"
+        "illuminate/console": "^8.0|^9.0|^10.0",
+        "illuminate/contracts": "^8.0|^9.0|^10.0",
+        "illuminate/filesystem": "^8.0|^9.0|^10.0",
+        "illuminate/routing": "^8.0|^9.0|^10.0",
+        "illuminate/support": "^8.0|^9.0|^10.0",
```

Comment:
Laravel 8 ondersteunt geen php 8.1 dus we gaan Laravel 8 support moeten droppen. Tests falen hierdoor ook